geolucidate/parser.py

```diff
@@ -29,9 +29,11 @@
     $
   """, re.VERBOSE | re.UNICODE | re.IGNORECASE)
 
-degree_min_sec_re = re.compile(r"""\b
+degree_min_sec_re = re.compile(r"""
+    ^
+    \(?
     # Latitude direction, first position: one of N, S, NORTH, SOUTH
-    ((?P<latdir>NORTH|SOUTH|[NS])\ ?)?
+    (((?P<latdir>NORTH|SOUTH|[NS])\ ?)|(?P<latsign>-))?
```

Comment:
Added support for `-` delimiter. They're called `latsign` and `longsign`. Used in  [_cleanup()](https://github.com/Avenza/geolucidate/pull/5/files#diff-ec126768a1469c3f7965134da04b11fcR49)